ADAL/src/public/mac/ADTokenCache.h

```diff
@@ -59,4 +59,26 @@
 - (BOOL)removeItem:(nonnull ADTokenCacheItem *)item
              error:(ADAuthenticationError * __nullable __autoreleasing * __nullable)error;
 
+/* Removes all token cache items for a specific client from the keychain.
+ */
+- (BOOL)removeAllForClientId:(NSString * __nonnull)clientId
+                       error:(ADAuthenticationError * __nullable __autoreleasing * __nullable)error;
+
+/* Removes all token cache items for a specific user and a specific clientId from the keychain
+ */
+- (BOOL)removeAllForUserId:(NSString * __nonnull)userId
+                  clientId:(NSString * __nonnull)clientId
+                     error:(ADAuthenticationError * __nullable __autoreleasing * __nullable)error;
+
+/*
+ Removes all token cache items for a specific user from the keychain with
+ either com.microsoft.adalcache shared group by default or the one provided in setDefaultKeychainGroup method.
```

Comment:
there's no keychain for macOS, so I think we should update this comment :)

Reply:
👍